engine/redis-v2/utils.go

```diff
@@ -9,3 +16,46 @@ func join(args ...string) string {
 func splits(n int, s string) []string {
 	return strings.SplitN(s, "/", n)
 }
+
+func isLuaScriptGone(err error) bool {
+	return strings.HasPrefix(err.Error(), "NOSCRIPT")
+}
+
+// GenTimerManagerID generates a consumer ID by local ipv4 address, current time and
+// a UUID.
+func genTimerManagerID() (string, error) {
+	ipv4, err := getLocalIPv4()
+	if err != nil {
+		return "", err
+	}
+	currentMilliSec := time.Now().UnixNano() / int64(time.Millisecond)
+	return fmt.Sprintf("{%d}-{%s}-{%s}", currentMilliSec, ipv4.String(), uuid.GenUniqueID()), nil
+}
+
+func getLocalIPv4() (net.IP, error) {
```

Comment:
seems public or private IP was ok here since we didn't use it to communicate between instances.  The current implementation was internal local IP instead of local.

Reply:
fix, remove internal ip filter code